HW6/mergeList.h

```diff
@@ -0,0 +1,25 @@
+#include <stdbool.h>
+#ifndef HOMEWORKS_MERGELIST_H
+#define HOMEWORKS_MERGELIST_H
+#define maxLength 256
+typedef int Error;
+
+typedef struct List
+{
+    char name[maxLength];
+    char phone[maxLength];
+    struct List *next;
+} List;
+// заполнение списка значениями из файла
+Error fillList(List **head, char *pathToFile);
```

Comment:
Если функция не собирается менять параметр, его имеет смысл передавать как const (и вообще писать const везде, где только можно):
```suggestion
Error fillList(List **head, const char * const pathToFile);
```